### DISCUSSION TITLE
The size of the WSL distribution backup has increased significantly

### DISCUSSION THREAD

**Discussion by TomYork1986** [microsoft/WSL#8087] [General]
Hello, 
I am facing the following problem: After upgrading to WSL2 backup file size has increased significantly (3-4Gb -> 58Gb). The size of a VHDX file is about 5.8Gb. 

Has anyone encountered a similar problem?

**Comment by Biswa96** [comment]
It sounds like this issue https://github.com/microsoft/WSL/issues/4699. If the actual file system in that VHDX file is smaller it is possible to shrink with `optimize-vhd` cmdlet in Powershell.

**Reply by TomYork1986** [reply]
Not really, I have no problem with the vhdx size, but with the tar file after the wsl --export command.
Anyway, I tried to run optimize-vhd command and this didn't help.
